pkg/plugins/ifaces.go

```diff
@@ -134,6 +134,11 @@ type RoleRegistry interface {
 	DeclarePluginRoles(ctx context.Context, ID, name string, registrations []RoleRegistration) error
 }
 
+// ActionSetRegistry handles the plugin RBAC actionsets
+type ActionSetRegistry interface {
```

Comment:
Unless you need to use this interface or any of its models within `pkg/plugins/*`, then I think this could actually be moved to `pluginsintegration` instead. If we only need to rely on this as a pipeline step (which looks to be the case), then we can consider it a "Grafana server" specific detail. 

Maybe the same can be said for `RoleRegistration`, but that's for another day 😄

Reply:
Good point, I just stuck it next to `RoleRegistration`. I'll see if I can move both of them (if it's too big a change, I might do it in a separate PR).